tools/experiments/experiments.js

```diff
@@ -138,6 +143,34 @@ function build() {
   } else {
     redirect.setAttribute('hidden', '');
   }
+
+  const rtvInput = document.getElementById('rtv');
+  const rtvButton = document.getElementById('rtv-submit');
+  rtvInput.addEventListener('input', () => {
+    rtvButton.disabled = !(
+      rtvInput.value == '' || RTV_PATTERN.test(rtvInput.value)
+    );
+    rtvButton.textContent = rtvInput.value == '' ? 'opt-out' : 'opt-in';
+  });
+  rtvButton.addEventListener('click', () => {
+    if (rtvInput.value == '') {
+      showConfirmation_(
+        'Do you really want to opt OUT of specific RTV',
+        setAmpCanaryCookie_.bind(null, AMP_CANARY_COOKIE.DISABLED)
+      );
+    } else if (RTV_PATTERN.test(rtvInput.value)) {
+      showConfirmation_(
+        `Do you really want to opt into the specific RTV ${rtvInput.value}?`,
+        setAmpCanaryCookie_.bind(null, rtvInput.value)
+      );
+    }
+  });
+
+  if (isExperimentOn_(RTV_EXPERIMENT_ID)) {
+    rtvInput.value = getCookie(window, 'AMP_CANARY');
+    rtvInput.dispatchEvent(new Event('input'));
+    document.getElementById('rtv-details').open = true;
```

Comment:
What is this doing?

Reply:
If you are already opted-in to a specific RTV it opens the section (as if you clicked on the right-pointing triangle)